tests/test_monitor.py

```diff
@@ -127,11 +127,25 @@ def test_new_clients_appear_in_focused_monitor(hlwm):
     assert hlwm.get_attr('clients', winid, 'tag') == 'tag2'
 
 
-def test_detect_monitors_does_not_crash(hlwm):
+@pytest.mark.parametrize("arg", ['-l', '--list-all', '--no-disjoin'])
+def test_detect_monitors_does_not_crash(hlwm, arg):
     # I don't know how to test detect_monitors properly, so just check that
     # it does not crash at least
+    hlwm.call(['detect_monitors', arg])
+
+
+def test_detect_monitors_root_window(hlwm):
```

Comment:
I don't quite get this test name. Isn't it more something `test_detect_monitors_has_effect_on_monitors_list` or `test_detect_monitors_yields_correct_monitors_list`?

Reply:
Yes, (see other comment). I've updated the name.